Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -240,6 +241,32 @@ public class PBXProjGenerator {
         var copyWatchReferences: [String] = []
         var extensions: [String] = []
 
+        // Find bundles within a targets sources
+        let bundlePaths = target.sources
```

Comment:
It seems bundles would then only be supported if they are added explicitly as a single source, not if they lived inside a sources directory?

Reply:
You're right! Bundles should be added to targets, if they are contained within a sources directory.